### PR TITLE
Revendor MAO

### PR DESCRIPTION
**Pull request by alexander-demichev**
Revendor MAO for dropping the requirement to set cluster ID label manually.

https://github.com/openshift/machine-api-operator/pull/608

### CODE REVIEW

File: pkg/termination/handler_test.go [hunk 1]
```diff
@@ -102,7 +102,7 @@ var _ = Describe("Handler Suite", func() {
 		var testMachine *machinev1.Machine
```
Comment (enxebre) [inline pkg/termination/handler_test.go:103]: why is this change needed now?

Comment (alexander-demichev) [reply pkg/termination/handler_test.go:103]: It seems to me that new version on envtest requires pre-created test namespace, otherwise unit tests fail

Comment (enxebre) [reply pkg/termination/handler_test.go:103]: it'd be nice to specify that in the commit desc

**Comment by enxebre** [comment]
also https://github.com/openshift/cluster-api-provider-gcp/pull/98#discussion_r441454340
